Cartfile

```diff
@@ -1,4 +1,3 @@
 github "the0neyouseek/MediaKeyTap" "master"
-github "reitermarkus/DDC.swift" "master"
```

Comment:
Is there a specific reason why you removed the Carthage dependency here and opted to just include the source? 

It's also throwing an error for me in the import in AppDelegate (No such module 'DDC'). 

Regardless I don't think we should just replace the carthage dependency with a source file as it decreases maintainability.

Reply:
I see the issue now, it's not working properly without doing it this way.. Still, we should probably look into getting it fixed instead of just manually including it.